src/components/network/network.js

```diff
@@ -152,6 +152,10 @@ genotet.NetworkView = function(viewName, params) {
     .on('genotet.nodeClick', function(event, node) {
       this.panel.displayNodeInfo(node);
       this.loader.incidentEdges(node);
+      this.signal('link', {
```

Comment:
The view should work in a way that is COMPLETELY unaware of links. So views can just fire clicks, and listen for signals like 'addProfiles'. If you ever want to write 'link' in the code of a view, you are probably doing sth inproperly.